packages/e2e/test/web-extension/dapp-sdk/features/singleUndelegation.ts

```diff
@@ -0,0 +1,32 @@
+import { ObservableWallet } from '@cardano-sdk/wallet';
+import { buildAndSignTx } from '../utils';
+import { combineLatest, take, tap } from 'rxjs';
+
+export const singleUndelegation = ({
+  connectedWallet,
+  logger
+}: {
+  logger: typeof console;
+  connectedWallet: ObservableWallet;
+}) => {
+  const undelegateAssetsElement = document.querySelector('#info-undelegate-assets')!;
+
+  combineLatest([connectedWallet.balance.utxo.available$])
```

Comment:
This `combineLatest` is redundant, since we're using only 1 observable
```suggestion
  connectedWallet.balance.utxo.available$
```

Reply:
it's addressed here: ac7ca1e3962ca8240a8e701b545712cc53127eaa